extensions/wikia/AdEngine/js/mercury.run.js

```diff
@@ -1,5 +1,14 @@
 /*global require*/
-require(['ext.wikia.adEngine.messageListener'], function (messageListener) {
+require([
+	'ext.wikia.adEngine.messageListener',
+	'ext.wikia.adEngine.customAdsLoader'
```

Comment:
Yeap, alphabetical order is better, but not a biggie.

On 03/24/2015 06:29 PM, Igor Rogatty wrote:
> In extensions/wikia/AdEngine/js/mercury.run.js
> <https://github.com/Wikia/app/pull/6758#discussion_r27052067>:
>
>> @@ -1,5 +1,14 @@
>>  /*global require*/
>> -require(['ext.wikia.adEngine.messageListener'], function (messageListener) {
>> +require([
>> +	'ext.wikia.adEngine.messageListener',
>> +	'ext.wikia.adEngine.customAdsLoader'
>
> Shouldn't it be kept in an alphabetic order?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Wikia/app/pull/6758/files#r27052067>.
>